### ISSUE TITLE
General doubt

### ISSUE THREAD

**Issue by jcgarciaca** [cosmicad/dataset#1]
Hi @cosmicad 

I am trying to use my dataset (it is just a small set) [https://drive.google.com/open?id=1xc0hAg1Ci7qDtw-wtV45y4wGx3TBagCZ](url) with YOLO and using the code in [https://github.com/experiencor/basic-yolo-keras](url). I read your issue there.

I am wondering if maybe you could help me. While training stage the loss value is continue decreasing, in 16th epoch it reaches around 0.09 although val_loss is really higher (around 0.3). Nevertheless, when I try to predict some images from training dataset, it doesn't find any box. I think it is so strange.

I suppose the anchors value must be adjusted to achieve good results. Did you do this?

Any advice is welcome.

Thank you.

**Comment by jcgarciaca** [comment]
I tested your dataset (with initial anchors) and training process started with loss: 0.9... meanwhile mine started with 14. By the end of first epoch with your dataset I get 18 boxes found
